package.json

```diff
@@ -1,7 +1,18 @@
 {
-  "name": "design-system-prototypes",
-  "private": true,
   "version": "0.0.0",
+  "author": "City of Amsterdam",
+  "description": "Prototyping environment for the Amsterdam Design System",
+  "license": "EUPL-1.2",
+  "name": "@amsterdam/design-system-prototypes",
+  "keywords": [
+    "nl-design-system"
+  ],
```

Comment:
Do we want to keep this? This repo is not really part of the NLDS ecosystem imo

Reply:
I thought it could be a showcase for it, but removed it as well.